src/content/docs/start/project-structure.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Project Structure
+i18nReady: true
+---
+
+A Tauri project is usually made of 2 parts, a Rust project and a JavaScript project (optional),
+and typically the setup looks something like this:
+
+```
+.
+├── package.json
+├── index.html
+├── src/
+│   ├── main.js
+├── src-tauri/
+│   ├── Cargo.toml
+│   ├── Cargo.lock
+│   ├── build.rs
+│   ├── tauri.conf.json
+│   ├── src/
+│   │   ├── main.rs
+│   │   └── lib.rs
+│   ├── icons/
+│   │   ├── icon.png
+│   │   ├── icon.icns
+│   │   └── icon.ico
+│   └── capabilities/
+│       └── default.json
+```
+
+In this case, the JavaScript project is at the top level, and the Rust project is inside `src-tauri/`,
+the Rust project is a normal [Cargo project](https://doc.rust-lang.org/cargo/guide/project-layout.html) with some extra files:
+
+- `tauri.conf.json` is the main configuration file for Tauri, it contains everything from the application identifier to dev server url,
+  this file is also a marker for the [Tauri CLI](/reference/cli/) to find the Rust project,
+  to learn more about it, see [Tauri Config](/develop/configuration-files/#tauri-config)
+- `capabilities/` directory is the default folder Tauri reads [capability files](/security/capabilities/) from (in short, you need to allow commands here to use them in your JavaScript code),
+  to learn more about it, see [Security](/security/)
+- `icons/` directory is the default output directory of the [`tauri icon`](/reference/cli/#icon) command, it's usually referenced in `tauri.conf.json > bundle > icon` and used for the app's icons
+- `build.rs` contains `tauri_build::build()` which is used for tauri's build system
+- `src/lib.rs` contains the Rust code and the mobile entry point, the reason we don't write directly in `main.rs` is because we compile your app to
+  a library in mobile builds and load them through the platform frameworks
+- `src/main.rs` is the main entry point for the Desktop
```

Comment:
we should probably specify that this file usually don't get modified. it's quite common in #support to see people modify both lib.rs and main.rs and then wonder why it's not working as expected.

Reply:
Don't modify `main.rs`, and modify `lib.rs` instead right? Made some explanations, don't know if it's good though 😂